docs/design/autoware-architecture/.pages

```diff
@@ -0,0 +1,10 @@
+nav:
+  - index.md
+  - control
+  - localization
+  - perception
+  - planning
+  - sensing
+  - vehicle
+  - map
```

Comment:
Did you try to order components alphabetically? :thinking: 
Or are there any reasons for putting `map` at the end?

```suggestion
  - control
  - localization
  - map
  - perception
  - planning
  - sensing
  - vehicle
```

Reply:
Or use processing order? But I'm not sure which is the former, `localization` or `perception`.

```suggestion
  - sensing
  - map
  - localization
  - perception
  - planning
  - control
  - vehicle
```

Reply:
I prefer the processing order.

Reply:
Done in https://github.com/autowarefoundation/autoware-documentation/pull/86/commits/b9a3f80b26b99714f8b16cb16038677b970bc1ec (for processing order)